.github/workflows/test.yml

```
@@ -119,11 +135,15 @@ jobs:
       - name: Add build artifacts directory to PATH for windows
         if: matrix.flavor.target == 'windows' && ! github.event.pull_request.draft
         shell: pwsh
-        run: |
+        run: |-
```

actionlint (1.7.7)

format string "\"%s\"" does not contain placeholder {0}. remove argument which is unused in the format string

(expression)